modules/mobile/spec/models/adapters/appointment_requests_adapter_spec.rb

```diff
@@ -328,12 +328,49 @@
           address: { street: '123 Sesame St.', city: 'Cheyenne', state: 'VA', zip_code: '20171' },
           lat: nil,
           long: nil,
-          phone: { area_code: '703', number: '652-0000', extension: nil },
+          phone: { area_code: '703', number: '652-0000', extension: 'x1000' },
           url: nil,
           code: nil
         }
         expect(adapted_cc_appt_request.location.to_h).to eq(expected_location)
       end
+
+      it 'handles nil phone numbers' do
```

Comment:
My next ticket is to extract the phone number parser into a helper method. It will involve detailed testing.